honeybadgermpc/sequencer.py

```diff
@@ -0,0 +1,25 @@
+import heapq
+
+
+class Sequencer(object):
```

Comment:
Nice, this makes sense. It's an abstract data structure implemented just on top of a priority queue. It holds items of the form `(i, val)`. Items can be enqueued in any order, but `get` will always return them in strict counting order, `(1,_),(2,_),...` so values added out of order are buffered until the next counting number is available